platform/ui/src/components/Icon/getIcon.jsx

```diff
@@ -21,11 +22,20 @@ import sorting from './../../assets/icons/sorting.svg';
 import sortingActiveDown from './../../assets/icons/sorting-active-down.svg';
 import sortingActiveUp from './../../assets/icons/sorting-active-up.svg';
 
+/** Tools */
+import toolZoom from './../../assets/icons/tool-zoom.svg';
+import toolCapture from './../../assets/icons/tool-capture.svg';
+import toolLayout from './../../assets/icons/tool-layout.svg';
+import toolMore from './../../assets/icons/tool-more-menu.svg';
+import toolMove from './../../assets/icons/tool-move.svg';
+import toolWindow from './../../assets/icons/tool-window-level.svg';
```

Comment:
I can't think of better names at the moment. I think the `tool` prefix works for now 👍